public/js/lib/themes/light-theme.css

```diff
@@ -252,3 +257,94 @@ div.CodeMirror span.eval-text {
   line-height: 1.4em;
   min-height: 1.4em;
 }
+
```

Comment:
Could you comment out or include the png urls?

https://dxr.mozilla.org/mozilla-central/source/devtools/client/themes/light-theme.css#267

This way we'll either have them or be able to add them later